src/main/java/org/apache/pulsar/io/sftp/source/SFTPSourceConfig.java

```diff
@@ -120,6 +124,17 @@ public class SFTPSourceConfig implements Serializable {
      */
     private Integer numWorkers = 1;
 
+    /**
+     * Used to distribute synchronization tasks,
+     * The producer has only one instance listening to the directory through WaitForExclusive mode.
+     */
+    private String sftpTaskTopic = "sftp_task";
```

Comment:
`sftpTaskTopic` needs to ensure that it does not have the same name as `destination-topic-name` in source create  command